R/rdeephaven/R/update_by_ops_wrapper.R

```diff
@@ -150,18 +782,77 @@ uby_ems_tick <- function(decay_ticks, cols = character(), operation_control = op
   return(UpdateByOp$new(INTERNAL_ems_tick(decay_ticks, cols, operation_control$.internal_rcpp_object)))
 }
 
+#' @name
+#' uby_ems_time
+#' @title
+#' Exponential moving sum by group with time as the decay unit
+#' @md
+#'
 #' @description
 #' Creates an exponential moving sum (EMS) UpdateByOp for each column in `cols`, using time as the decay unit.
+#'
+#' @details
 #' The formula used is
-#'   `dt_current = current_timestamp - prev_timestamp`
-#'   `a_current = e^(-dt_current / decay_time)`
-#'   `ems_first = first_value`
-#'   `ems_current = a_current * ems_prev + current_value`
-#' @param ts_col String denoting the column to use as the timestamp.
-#' @param decay_time ISO-8601-formatted string specifying the decay rate.
+#' \deqn{a_i = e^{\frac{-dt_i}{\tau}}}
+#' \deqn{\mathcal{S}_0 = x_0}
+#' \deqn{\mathcal{S}_i = a_i*\mathcal{S}_{i-1} + x_i}
+#'
+#' Where:
+#' - \eqn{dt_i} is the difference between time \eqn{t_i} and \eqn{t_{i-1}} in nanoseconds.
+#' - \eqn{\tau} is `decay_time` in nanoseconds, an input parameter to the method.
```

Comment:
same here.  I'm guessing that there are other cases as well.  Could probably also remove "in nanoseconds" from the prior line.